mitiq/calibration/calibrator.py

```diff
@@ -70,7 +72,7 @@ def __init__(
             else None
         )
         self.settings = settings
-        self.circuits = self.settings.make_circuits()
+        self.circuits = self.settings.make_circuits(self.settings.circuit_seed)
```

Comment:
```suggestion
        self.circuits = settings.make_circuits(settings.circuit_seed)
```
This is a bit simpler and should be equivalent, if I am not wrong.

Reply:
This could also work, but I changed`make_circuits` so that it no longer has the seed as an explicit argument.